pkg/environment/environmentApply.go

```diff
@@ -112,8 +115,8 @@ func (a *Apply) Plan() error {
 // else checks for PR number and get the list of changed namespaces in that merged PR. Then does the kubectl apply and
 // terraform init and apply of all the namespaces merged in the PR
 func (a *Apply) Apply() error {
-	if a.Options.NMinutes == 0 && a.Options.Namespace == "" {
-		err := fmt.Errorf("either minutes or a namespace is required to perform apply")
+	if a.Options.CommitTimestamp == "" && a.Options.Namespace == "" {
+		err := fmt.Errorf("either commit timestamp or a namespace is required to perform apply")
```

Comment:
You could just return the error rather than assigning it to a var and returning the var.